cmd/internal/shared/gatewayserver/config.go

```diff
@@ -31,9 +35,11 @@ var DefaultGatewayServerConfig = gatewayserver.Config{
 			":1700": "",
 		},
 	},
-	MQTT: gatewayserver.MQTTConfig{
+	MQTT: config.MQTT{
 		Listen:    ":1882",
 		ListenTLS: ":8882",
+		Public:    fmt.Sprintf("mqtt://%s:1883", shared.DefaultPublicHost),
+		PublicTLS: fmt.Sprintf("mqtts://%s:8883", shared.DefaultPublicHost),
```

Comment:
@htdvisser should we use `Address` suffix, like we do with `URL` in other config?

Reply:
This is consistent with `Listen`, so we either add `Address` to both or leave it as is.

Reply:
Yes, that's true. OK, fine then!